### PR TITLE
App Submission: Fenrus Dashboard

### PR DESCRIPTION
**Pull request by awholegnuworld**
This is my first time using docker, please be patient. 😅 I don't know if I did this correctly.

This is not my app, it is just an app that I thought was cool.
https://github.com/revenz/Fenrus

Image for app store:

### CODE REVIEW

**Comment by lukechilds** [comment]
>This is my first time using docker, please be patient. 😅 I don't know if I did this correctly.

No worries!

Your `umbrel-app.yml` file looked great, however the `docker-compose.yml` file is currently just the template. You need to fill in the values to run a Fenrus Docker container and setup any required environment variables or volumes.

File: fenrus/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+version: "1.0"
```
Comment (lukechilds) [inline fenrus/docker-compose.yml:1]: This file needs to be populated.

**Review by lukechilds — CHANGES_REQUESTED** [review]

**Comment by nevets963** [comment]
Hey @awholegnuworld Is there something that can be done to help progress this PR? Just checking in to see if you're still working on it?

Steve

**Comment by awholegnuworld** [comment]
> Hey @awholegnuworld Is there something that can be done to help progress this PR? Just checking in to see if you're still working on it?
> 
> 
> 
> Steve

Thanks for checking in! Yeah, I'm still working on it, but school just started and I have other things to do right now. The main thing that needs to be done is filling in the docker file.

**Comment by awholegnuworld** [comment]
Ok, I'm sorry for waiting so long. I finally had some time off and remembered to work on this.

**Comment by awholegnuworld** [comment]
Can do.

**Comment by nmfretz** [comment]
Hey @awholegnuworld did you have any luck getting the developer to consider versioned Docker images? If not, would you still like to keep this PR open?

**Comment by nmfretz** [comment]
@awholegnuworld just checking in again to see if you would like to keep this PR open.

**Comment by awholegnuworld** [comment]
I'm fine to close the PR. I haven't been working with my Umbrel lately.

**Comment by nmfretz** [comment]
Thanks @awholegnuworld. Please open this again if you end up wanting to work on it!

**Pull request closed without merging**